### PR TITLE
support for custom st2actionrunner images via values.yaml

### PR DESCRIPTION
**Pull request by Sheshagiri**
This pr adds support for providing custom st2actionrunner-specific docker repository, image name, pull policy, and pull secret via `values.yaml`.

closes: #104

### CODE REVIEW

**Comment by Sheshagiri** [comment]
@armab this is ready for review the following todo's

- [ ] actual testing on a real K8s cluster
- [ ] update release notes and readme

File: templates/_helpers.tpl
Comment (armab) [inline templates/_helpers.tpl:34]: For simplicity reasons, it would be easier to avoid custom template for st2actionrunner, if possible. I saw the previous version, wondering if that worked good?

**Review by cognifloyd — COMMENTED** [review]
Is this PR still needed? If so, can you rebase it or merge in master?
Enterprise stuff has been removed, so at least some of this can be simplified.

**Comment by Sheshagiri** [comment]
@cognifloyd will take a look at this, this week.

**Comment by cognifloyd** [comment]
#200 was just merged, so you can have custom image tags for any of the pods, including `st2actionrunner`. Hopefully that simplifies things for you @Sheshagiri .

**Comment by cognifloyd** [comment]
@Sheshagiri I looked into resolving the conflicts for you to merge master into your branch, but Maintainer edits are not turned on.

Would you please merge in master? I'd love to merge this once we get it ready.

**Comment by cognifloyd** [comment]
@Sheshagiri I merged in master and addressed @armab's feedback by dropping the additional helper.
Does this do what you were hoping for?

**Review by cognifloyd — APPROVED** [review]
K. I added unit tests.

@armab would you please review this?
I changed quite a bit, so I need another pair of eyes before it gets merged.

**Comment by cognifloyd** [comment]
Yeah that's what I figure. Thanks!

**Pull request merged**
